src/types/domain.d.ts

```diff
@@ -293,13 +293,15 @@ export interface PublicServiceRule {
   name: Partial<TextLanguage>;
   description: Partial<TextLanguage>;
   language: PublicServiceLanguage[];
-  implements: PublicService[];
+  implements: string[];
 }
 
 export interface PublicServiceLegalResource {
-  uri: string;
+  dctTitle: Partial<TextLanguage>;
   description: Partial<TextLanguage>;
-  url: string;
+  relation: string[];
+  seeAlso: string[];
+  uri: string;
```

Comment:
var det for holde det alfabetisk at du flytter uri?

Reply:
Nei, skrev de inn med samme rekkefølge som de stod i json